docs/sample_config.yaml

```diff
@@ -142,6 +142,17 @@ presence:
 #
 #limit_profile_requests_to_users_who_share_rooms: true
 
+# User Directory Updates
+#
+# If specified, a worker with that name will be the only one able to update
+# the user directory. Important for querying shared rooms. If the worker is
+# specified, but not running, the user directory may become outdated.
+#
+# Defaults to null, meaning the main process will handle this.
+#
+# Set to false to make sure no process will handle the user_directory.
```

Comment:
why would you want *no* process to handle the user directory? why would that be useful?

Reply:
See [this comment](https://github.com/matrix-org/synapse/pull/11450#issuecomment-987109870)